app/screens/sidebar/cityFilter/cityFilter.less

```diff
@@ -0,0 +1,14 @@
+.city-filter {
+  width: 100%;
+  padding: 0.5rem;
+  background-color: @primary-color;
+  color: @primary-contrast-color;
+  border: none;
+  border-radius: 2px;
```

Comment:
- [ ] the default border radius size could be added to variables. Then it's easier to change if needed.